File: MODUL-2/Python/PRAK204-2310817220002-RaudatulSholehah.py
print("input")
Nilai = input().split()

if len(Nilai)==1:
    jari_jari = float(Nilai[0])
    Tinggi    = float(input())

elif len(Nilai)==2:
    jari_jari, Tinggi = map(float, Nilai)

v = 3.142857
volume   = v * jari_jari * jari_jari * Tinggi
luas     = 2 * v * jari_jari * (jari_jari + Tinggi)
keliling = 2 * v * jari_jari

print("output")
print(f"volume    = {volume:.2f}")
print(f"luas      = {luas:.2f}")
print(f"keliling  = {keliling:.2f}")

File: MODUL-2/C/PRAK204-2310817220002-RaudatulSholehah.c
#include <stdio.h>
#include <string.h>
#define v 3.142857
int main()
{
    float jari_jari, tinggi, volume, luas, keliling;
    printf("Input\n");
    scanf("%f %f", &jari_jari, &tinggi);

    volume = v * jari_jari * jari_jari * tinggi;
    luas = 2 * v * jari_jari * (jari_jari + tinggi);
    keliling = 2 * v * jari_jari;

    printf("Output\nvolume    = %.2f\n", volume);
    printf("luas      = %.2f\n", luas);
    printf("keliling  = %.2f\n", keliling);
    return 0;
}